### PR TITLE
fix(minor code defects):

### PR DESCRIPTION
**Pull request by msoedov**
Fix minor code defects, use module level re compiled PUNCTUATION_MATCHER instead of 3 different references, remove unused prints, and deprecate unused variables.

### CODE REVIEW

**Review by jclerman — COMMENTED** [review]
We should apply flake8 via a pre-commit hook.  That would have caught some of the issues fixed here.

File: philter_lite/philter.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def detect_phi(
     # create intersection maps for all phi types and add them to a dictionary containing all maps
     # get full exclude map (only updated either on-command by map_regex_context or at the very end of map_
     # coordinates)
-    full_exclude_map = include_map.get_complement(text_data)
+    # full_exclude_map = include_map.get_complement(text_data)
```
Comment (msoedov) [inline philter_lite/philter.py:121]: Unused function call, just commented out for now.

File: philter_lite/coordinate_map.py [hunk 2]
```diff
@@ -23,14 +25,13 @@ def __init__(self, pattern={"title": "untitled"}, debug=False):
 
         self.map = {}
         self.coord2pattern = {}
-        self.pattern = pattern
-        self.debug = debug
```
Comment (jclerman) [inline philter_lite/coordinate_map.py:27]: why were these removed?  This is a breaking change since it removes attributes from this object-type.

Comment (msoedov) [reply philter_lite/coordinate_map.py:27]: those are never set and unused

Comment (jclerman) [reply philter_lite/coordinate_map.py:27]: doesn't that depend on whether code calls them?

Comment (msoedov) [reply philter_lite/coordinate_map.py:27]: I have checked, and it was not used

Comment (jclerman) [reply philter_lite/coordinate_map.py:27]: Not used in our other consuming code, but other consuming code might reference those params/attributes (unknown to us).  So, this is a breaking change - now reflected in the new version number (0.5.0). All good.

**Review by jclerman — APPROVED** [review]
Looks good; thanks for these cleanup changes.
